<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountSettings</class>
 <widget class="QDialog" name="AccountSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>100</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>41</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="buddy">
    <cstring>lePass</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePass">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sync Mode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSync">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>180</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Manual</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Auto</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="leUser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewAccount">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;a href=&quot;http://wikenotes.com/index.php/user/register&quot;&gt;New Account&lt;/a&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leUser</tabstop>
  <tabstop>lePass</tabstop>
  <tabstop>cmbSync</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AccountSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AccountSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
